spellcheck.json

```diff
@@ -1,5 +1,9 @@
 {
   "witdh": "width",
   "hieght": "height",
-  "hight": "height"
+  "hight": "height",
+  "hmtl": "html",
+  "htlm": "html",
```

Comment:
Not sure about this one. https://www.google.com/search?q=%2Bhtlm&nfpr=1&sa=X
Seems like that might be intentional in some cases. We should maybe put that one in a different file.

Reply:
@ScolderCreations What do you think?

Reply:
Now that I think about it, intentional misspellings are ridiculous.